### PR TITLE
Ignore push events that delete branch

### PR DESCRIPTION
**Pull request by mfocko**
Signed-off-by: Matej Focko <mfocko@redhat.com>

### CODE REVIEW

File: tests/unit/test_steve.py [hunk 1]
```diff
@@ -108,3 +108,20 @@ def test_process_message(event, private, enabled_private_namespaces, success):
     )
     assert "propose_downstream" in next(iter(results["job"]))
     assert first_dict_value(results["job"])["success"]
+
+
+@pytest.fixture()
+def github_push():
+    with open(DATA_DIR / "webhooks" / "github" / "push.json") as outfile:
```
Comment (lachmanfrantisek) [inline tests/unit/test_steve.py:115]: We already have an example. Nice!

**Review by lachmanfrantisek — APPROVED** [review]
Looks good to me. Thank you!

File: tests/unit/test_steve.py (hunk 1 above)
Comment (mfocko) [reply tests/unit/test_steve.py:115]: that's also a problem :D it's copy-paste, would be nice to do a PR and reorganize all those fixtures

Comment (lachmanfrantisek) [reply tests/unit/test_steve.py:115]: Would be really nice!

**Pull request merged**